dle/data/management/commands/load_tga_data.py

```diff
@@ -0,0 +1,215 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.db import IntegrityError
+from requests.exceptions import ChunkedEncodingError
+from data.models import (
+    DrugLabel,
+    LabelProduct,
+    ProductSection,
+)
+from users.models import MyLabel
+from django.core.files.base import ContentFile
+from django.core.files.storage import default_storage
+from django.conf import settings
+import requests
+import fitz  # PyMuPDF
+from bs4 import BeautifulSoup
+import re
+import datetime
+import pandas as pd
+import time
+import logging
+import random
+import string
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+logger = logging.getLogger(__name__)
+
+TGA_BASE_URL = "https://www.ebs.tga.gov.au/ebs/picmi/picmirepository.nsf/"
+
+# runs with `python manage.py load_tga_data`
+# add `--type full` to import the full dataset
+# add `--verbosity 2` for info output
+# add `--verbosity 3` for debug output
+class Command(BaseCommand):
+    count = 0 # TODO: Remove once pdf parser is supported
+
+    help = "Loads data from TGA"
+
+    def __init__(self, stdout=None, stderr=None, no_color=False, force_color=False):
+        super().__init__(stdout, stderr, no_color, force_color)
+        self.num_drug_labels_parsed = 0
+        "keep track of the number of labels processed"
+        self.error_urls = {}
+        "dictionary to keep track of the urls that have parsing errors; form: {url: True}"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--type",
+            type=str,
+            help="'full'",
+            default="full",
```

Comment:
Do we want to add tests at this point? E.g. https://github.com/globalrx/globalrx/blob/27871ed2e56eb6935bc4aaacd0d27974892fc3b1/dle/data/tests.py#L81-L86

Reply:
Maybe wait til we have the parse support?

Reply:
https://cscie599-drug-labels.atlassian.net/browse/DL-107
Created a ticket to add tests